src/AppBundle/Controller/Admin/AdminCommitteeController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use AppBundle\Entity\Committee;
+use AppBundle\Exception\CommitteeException;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/committee")
```

Comment:
Then please remove it from https://github.com/EnMarche/en-marche.fr/blob/feat-admin-approve-committee/src/AppBundle/Controller/Admin/AdminAdherentController.php